docker_test_tools/environment.py

```diff
@@ -388,3 +388,10 @@ def get_container_id(self, name):
     def validate_service_name(self, name):
         if name not in self.services:
             raise ValueError('Invalid service name: %r, must be one of %s' % (name, self.services))
+
+    def run_exec_in_container(self, name, command):
```

Comment:
Please look at the other functions in this file that work on containers and make this one similar to them.
For example:
1. The other functions start with a docstring
2. The other functions start with a log.debug message